src/Workspaces/Core/Desktop/Workspace/SQLite/SQLitePersistentStorage.Accessor.cs

```diff
@@ -146,7 +147,7 @@ private Stream ReadBlob(SqlConnection connection, TDatabaseId dataId)
                     // data for a row in our system can change, the ID will always stay the
                     // same, and the data will always be valid for our ID.  So there is no
                     // safety issue here.
-                    return connection.ReadBlob(DataTableName, DataColumnName, rowId);
+                    return connection.ReadBlob(DataTableNamePtr, DataColumnName, rowId);
```

Comment:
shouldn't the ColumnName be a byte[] too?

Reply:
Yep, good catches so far. I haven't compiled locally, so this probably has issues even at compile time :p